app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -174,7 +174,7 @@ def storage_domain
       api :GET, "/compute_resources/:id/available_clusters/:cluster_id/available_storage_domains", N_("List storage domains for a compute resource")
       param :id, :identifier, :required => true
       param :cluster_id, String
-      param :storage_domain, String
+      param :storage_domain, String, deprecated: true
```

Comment:
This implies the endpoint listed at line 173 should also be considered deprecated because there's no other way to pass in the `storage_domain`.